app/assets/javascripts/hyrax/collections.js

```diff
@@ -10,6 +10,13 @@ Blacklight.onLoad(function () {
       form.append('<input type="hidden" value="add" name="collection[members]"></input>');
   });
 
+  // background banner image uploaded for branding  
+  $(".hyc-header header[data-background]").each(function() {
```

Comment:
I'd prefer we use attribute selectors like `data-background` for javascript behaviors and not classes. That way classes can be easily changed without affecting behaviors.

Reply:
Okay, I will update this to use an attribute selector.

Reply:
✅
I updated the show code to display the inline style directly. As such, I removed the redundant JS code you were referring to. There is now no selector to update. I will make note of your selector preference for future reference.